internal/responses/responses.go

```diff
@@ -47,3 +47,10 @@ type Sync TargetOperation
 type TargetOperation struct {
 	WorkflowName string `json:"workflow_name"`
 }
+
+// TokenResponse represents the output for CreateToken.
+type TokenResponse struct {
```

Comment:
Should this be named `CreateToken`, keeping in line with how the other structs are named?